replication/binlogsyncer.go

```diff
@@ -733,33 +733,53 @@ func (b *BinlogSyncer) parseEvent(s *BinlogStreamer, data []byte) error {
 		// Some events like FormatDescriptionEvent return 0, ignore.
 		b.nextPos.Pos = e.Header.LogPos
 	}
+
+	getCurrentGtidSet := func() GTIDSet {
```

Comment:
essentially moved `getGtidSet` to local function to emphasize that it is being used only inside this function